posthog/queries/test/test_stickiness.py

```diff
@@ -340,6 +340,29 @@ def test_stickiness_people_paginated(self):
             second_result = self.client.get(result["next"]).json()
             self.assertEqual(len(second_result["results"][0]["people"]), 50)
 
+        def test_stickiness_compared(self):
+            self._create_multiple_people()
+
+            filter = StickinessFilter(
+                data={
+                    "shown_as": "Stickiness",
+                    "date_from": "2020-01-01",
+                    "date_to": "2020-01-08",
+                    "compare": "true",
+                    "display": "ActionsLineGraph",
+                    "events": '[{"id":"watched movie","math":"dau","name":"watched movie","type":"events","order":null,"properties":[],"math_property":null}]',
+                    "insight": "TRENDS",
+                    "interval": "day",
+                    "properties": "[]",
+                    "shown_as": "Stickiness",
+                },
+                team=self.team,
+                get_earliest_timestamp=get_earliest_timestamp,
+            )
+            response = stickiness().run(filter, self.team)
+            self.assertEqual(response[0]["data"], [2, 1, 1, 0, 0, 0, 0, 0])
+            self.assertEqual(response[1]["data"], [3, 0, 0, 0, 0, 0, 0, 0])
```

Comment:
Is there a more meaningful test we could add here @EDsCODE? I just added this because it made sure it doesn't blow up.

Reply:
Could you also adjust it so that `TestStickiness` implements `AbstractCompareTest`? Also, I think an additional test could just be testing the `determine_compared_filter` directly. Will be nice to have some coverage on these helper functions that are reused in various contexts